digitalocean/resource_digitalocean_kubernetes_cluster_test.go

```diff
@@ -187,7 +187,7 @@ func testAccDigitalOceanKubernetesConfigBasic(rName string) string {
 resource "digitalocean_kubernetes_cluster" "foobar" {
 	name    = "%s"
 	region  = "lon1"
-	version = "1.15.3-do.1"
+	version = "1.15.3-do.3"
```

Comment:
@andrewsomething this won't work as-is, I opened another PR to deal with this bump here: https://github.com/terraform-providers/terraform-provider-digitalocean/pull/309

Reply:
Mind if I rebase it out and merge?

Reply:
Reverted, will merge on green.

Though all of the tests fail without it:

```
        Error: Error creating Kubernetes cluster: POST https://api.digitalocean.com/v2/kubernetes/clusters: 422 rpc error: code = InvalidArgument desc = validation error: invalid version slug
```